### PR TITLE
VONK-2871: Document the restriction of the order of a Facade plugin

### PR DESCRIPTION
**Pull request by cheng13231**

### CODE REVIEW

File: facade/finalizesearch.rst [hunk 1]
```diff
@@ -31,6 +31,11 @@ the order of inclusion, and adds to the services. For background information, se
         }
     }
 
+.. note::
+  As of Firely Server version 4.4.0, your Facade should not have an order greater than or equal to 211. 
+  The reason for this is that the MemoryAdministrationConfiguration (order: 211) used by the admin endpoint
```
Comment (cknaap) [inline facade/finalizesearch.rst:36]: Most people probably don't use the `MemoryAdministrationConfiguration`, so I would put it a bit more generic like: The reason for this is that upon configuring the administration database, Firely Server checks whether an ``ISearchRepository`` is registered. The earliest of these configurations is at order 211.

File: releasenotes/releasenotes.rst [hunk 2]
```diff
@@ -244,6 +244,7 @@ Plugin and Facade
 #. The interfaces PrioritizedResourceResolver(R3|R4|R5) and their implementations are no longer available. It is advised to construct your own StructureDefinitionSummaryProvider incl. a MultiResolver combining your own resource resolver and the IConformanceCache provided by Firely Server.
 #. The interface IConformanceCacheInvalidation has been moved from Vonk.Core.Import to Vonk.Core.Conformance
 #. The classes SpecificationZipResolver(R3|R4|R5) are no longer available. Please use the IPrioritizedResourceResolvers instead.
+#. Starting from this version, a Facade should not have an order greater than or equal to 211. The reason for this is that the MemoryAdministrationConfiguration (order: 211) used by the admin endpoint tries to resolve the ISearchRepository.
```
Comment (cknaap) [inline releasenotes/releasenotes.rst:247]: See above.

**Review by cknaap — CHANGES_REQUESTED** [review]

File: facade/finalizesearch.rst (hunk 1 above)
Comment (cheng13231) [reply facade/finalizesearch.rst]: Thanks. 👍

File: releasenotes/releasenotes.rst (hunk 2 above)
Comment (cheng13231) [reply releasenotes/releasenotes.rst]: Updated.

**Review by cknaap — APPROVED** [review]

**Pull request merged**